values.yaml

```diff
@@ -36,7 +36,17 @@ host_network:
       metricServer:
         port: 45056
 
-
+node_ports:
+  enabled: placeholder_enable_kubeadm_cluster
+  # the chosen http/https values are the same with LoadBalancer values 
+  oauth2_proxy:
```

Comment:
@hamzabouissi is there code for ansible that reserves the first 100 for glueops ports specifically?

Reply:
Got it. Just created a ticket to track it here: https://github.com/orgs/internal-GlueOps/projects/3/views/9?pane=issue&itemId=113033409&issue=internal-GlueOps%7Cissues%7C192